### PR TITLE
Update CSCC setup to reflect Terraform based install path

### PR DESCRIPTION
**Pull request by hshin-g**
Resolves Issue #2981.    

--

Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [ ] All of the [tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) pass
- [ ] I have submitted a corresponding PR in the [Forseti Terraform module](https://github.com/forseti-security/terraform-google-forseti) (if applicable).

These guidelines and more can be found in our [contributing guidelines](https://github.com/forseti-security/forseti-security/blob/dev/.github/CONTRIBUTING.md).

### CODE REVIEW

File: _docs/_latest/configure/notifier/index.md [hunk 1]
```diff
@@ -234,24 +234,18 @@ the UI or API:
 - Note: You will need to have either owner, editor or service management roles
   in the Project in order to enable the API
 
-1. Enable the  API connector config to Cloud SCC.  Specifically, this means
-in the Forseti project server bucket, edit the `configs/forseti_conf_server.yaml`,
- as follows:
+1. Enable the  API connector config to Cloud SCC by configuring the following fields in
```
Comment (joecheuk) [inline _docs/_latest/configure/notifier/index.md]: What about the path for using deployment manager?

File: _docs/_latest/configure/notifier/index.md [hunk 2]
```diff
@@ -234,24 +234,18 @@ the UI or API:
...
-  * `enabled:`
+  * `cscc_violations_enabled`
     * **Description**: Whether to send notification to Cloud SCC.
-    * **Valid values**: one of valid `true` or `false`
+    * **Valid values**: One of valid `true` or `false`
   
-  * `source_id`
+  * `cscc_source_id`
     * **Description**: ID from the Cloud SCC beta on-boarding. **This must be added**
     to use the Beta API integration.
```
Comment (blueandgold) [inline _docs/_latest/configure/notifier/index.md]: Can you please remove any mention of `Beta` for CSCC, while you are here?  CSCC is GA now.  Please look at lines 227, 230 245, 246, and 250

File: _docs/_latest/configure/notifier/index.md (hunk 1 above)
Comment (hshin-g) [reply _docs/_latest/configure/notifier/index.md]: Good point- please take look at updated.

File: _docs/_latest/configure/notifier/index.md (hunk 2 above)
Comment (hshin-g) [reply _docs/_latest/configure/notifier/index.md]: Fixed!

**Pull request merged**